pom.xml

```diff
@@ -64,9 +64,14 @@
     <maven.version>3.8.8</maven.version>
 
     <flink.version>1.14.6</flink.version>
-    <hadoop.version>3.2.4</hadoop.version>
     <spark.version>3.3.2</spark.version>
 
+    <!-- use hadoop-3 as default  -->
+    <hadoop.version>3.2.4</hadoop.version>
```

Comment:
Curious, why `3.2.4` not the last release `3.3.6`?

Reply:
Hadoop client 3.3.6 can not access hadoop 2.x HDFS. So we can not use hadoop 3.3.6 now.

Reply:
thanks for the explanation.